pkg/controller/certificates/util.go

```diff
@@ -102,6 +108,11 @@ func certificateMatchesSpec(crt *v1alpha2.Certificate, key crypto.Signer, cert *
 		errs = append(errs, fmt.Sprintf("DNS names on TLS certificate not up to date: %q", cert.DNSNames))
 	}
 
+	expectedURIs := crt.Spec.URISANs
+	if !util.EqualUnsorted(pki.URLsToString(cert.URIs), expectedURIs) {
+		errs = append(errs, fmt.Sprintf("URI SANs on TLS certificate not up to date: %q", cert.URIs))
+	}
```

Comment:
Can we get corresponding tests for these changes? I'm pretty sure this area of code is already unit tested